config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP permissions policy. For further
+# information see https://developers.google.com/web/updates/2018/06/feature-policy
+#
+Rails.application.config.permissions_policy do |f|
+  f.camera      :none
+  f.gyroscope   :none
+  f.microphone  :none
+  f.usb         :none
+  f.fullscreen  '*'
+  f.payment     :self
+end
```

Comment:
this is cool. i didnt know this was in rails 7

Reply:
Yeah, I find it pretty cool too!

Reply:
News to me too! I was like _SGTM!_.